### PR TITLE
updates from cob4-2

### PR DESCRIPTION
**Pull request by ipa-cob4-2**

### CODE REVIEW

File: cob_bringup/robots/cob4-2.xml [hunk 1]
```diff
@@ -93,7 +93,7 @@
 		</include>
 
 		<!-- startup hardware -->
-		<!--include file="$(find cob_bringup)/drivers/canopen_402.launch" >
+		<include file="$(find cob_bringup)/drivers/canopen_402.launch" >
 			<arg name="robot" value="$(arg robot)" />
 			<arg name="component_name" value="torso" />
 			<arg name="can_device" value="can1" />
```
Comment (fmessmer) [inline cob_bringup/robots/cob4-2.xml]: `can0` or `can1`?

Comment (floweisshardt) [inline cob_bringup/robots/cob4-2.xml]: Torso is on can0 and sensorring on can1. Seems as either a merge went wrong or not everything from the robot is committed and pushed.
@ipa-nhg please check

Comment (NadiaHG) [inline cob_bringup/robots/cob4-2.xml]: torso: can0

**Comment by floweisshardt** [comment]
LGTM

**Pull request merged**